addons/website_blog/migrations/9.0.1.0/post-migration.py

```diff
@@ -27,11 +27,10 @@ def convert_blog_post_cover(env):
         post = post_obj.browse(row[0])
         post.cover_properties = (
             '{{"background-image": "url({})",'
-            ' "opacity": "1",'
+            ' "opacity": "0",'
```

Comment:
Yeah, we should add another migration script in website_blog for v10, as not everybody makes v8 > v10 directly.